### ISSUE TITLE
🐛 Extension v3 doesn't work

### ISSUE THREAD

**Issue by vladkrasn** [biomejs/biome-vscode#614]
### VS Code version

1.100.2

### Extension version

3.0.0

### Biome version

1.9.4

### Operating system

- [x] Windows
- [ ] macOS
- [ ] Linux

### Description

After updating the extension to v3, It doesn't work for me on two separate windows systems. Steps from https://biomejs.dev/reference/vscode/#migrating-from-the-2x-extension didn't help.

Neither format on save nor linter highlighting errors works. In the bottom right corner, Biome icon is present and is not red. Extension logs don't show any errors. CLI works as expected

### Steps to reproduce

Trying to work with extension in any capacity. Check out reproduction.

### Expected behavior

Extension works better than v2!

### Does this issue occur when using the CLI directly?

No

### Link to a minimal reproduction

https://github.com/vladkrasn/biome-issue

### Logs
These kinds of traces do get put out.

```plain text
-05-29 15:38:49.484 [info] [Trace - 5:38:49 PM] Sending request 'textDocument/codeAction - (32)'.
2025-05-29 15:38:49.484 [info] Params: {
    "textDocument": {
        "uri": "file:///c%3A/Users/1234/Documents/test/biome-issue/src/index.ts"
    },
    "range": {
        "start": {
            "line": 0,
            "character": 0
        },
        "end": {
            "line": 5,
            "character": 0
        }
    },
    "context": {
        "diagnostics": [
            {
                "range": {
                    "start": {
                        "line": 1,
                        "character": 4
                    },
                    "end": {
                        "line": 1,
                        "character": 5
                    }
                },
                "message": "Variable 'a' implicitly has an 'any' type, but a better type may be inferred from usage.",
                "code": 7043,
                "severity": 4,
                "source": "ts"
            }
        ],
        "only": [
            "quickfix.biome"
        ],
        "triggerKind": 2
    }
}
```

**Comment by nhedger** [comment]
Came to the same conclusion. The current `include` makes the LSP server ignore your `index.ts` file.

Try this instead:

```diff
"files": {
    "include": [
-       "./src"
+       "./src/**/*"
    ]
},
```

I'll close the issue since it's not specific to the VS Code extension.

**Comment by nhedger** [comment]
Fixed in d02a61d0ef8d211db3394046e51962a711d153cc.

Released as v3.0.1 (available in a couple of minutes).
